packages/tokens/src/spacing.ts

```diff
@@ -9,6 +9,7 @@ const spacing = {
   400: 16,
   600: 24,
   800: 32,
+  900: 36,
```

Comment:
Noticed this after I made the ticket, but I think we also need 1000 and 1400

Reply:
Added!